supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/statuspolling/JesApiQueryManager.scala

```diff
@@ -72,44 +93,77 @@ class JesApiQueryManager(val qps: Int) extends Actor with ActorLogging {
   }
 
   private def handleTerminated(terminee: ActorRef) = {
-    // Currently we can assume this is a polling actor. Might change in a future update:
+    val cause = getFailureCause(terminee).getOrElse(new RuntimeException("No failure reason recorded"))
+    // We assume this is a polling actor. Might change in a future update:
```

Comment:
Ah right, I forgot about the reason for this in the first place, the cap on retries